tests/retriever/test_sequential_retriever.py

```diff
@@ -0,0 +1,63 @@
+"""Unit tests for the SequentialRetriever.
+
+Author: John Lambert
+"""
+
+import unittest
+from pathlib import Path
+
+from gtsfm.loader.olsson_loader import OlssonLoader
+from gtsfm.retriever.sequential_retriever import SequentialRetriever
+
+DATA_ROOT_PATH = Path(__file__).resolve().parent.parent / "data"
+DEFAULT_FOLDER = DATA_ROOT_PATH / "set1_lund_door"
+
+
+class TestSequentialRetriever(unittest.TestCase):
+    def test_sequential_retriever(self) -> None:
+        """Assert that we get 30 total matches with a lookahead of 3 frames on the Door Dataset."""
+
+        max_frame_lookahead = 3
+        loader = OlssonLoader(str(DEFAULT_FOLDER), image_extension="JPG", max_frame_lookahead=max_frame_lookahead)
```

Comment:
@johnwlambert why do we still need this lookahead parameter now?

Reply:
We need it because sequential matching needs a specification of the sliding window size.

Reply:
Yes, but why is that not just a retriever parameter?

Reply:
You're right, good catch. Can be removed. I recommend to do it in a small follow-up PR.